cmd/metrics.go

```diff
@@ -497,10 +528,50 @@ func storageMetricsPrometheus(ch chan<- prometheus.Metric) {
 	onlineDisks, offlineDisks := getOnlineOfflineDisksStats(server.Disks)
 	totalDisks := offlineDisks.Merge(onlineDisks)
 
+	// Report total capacity
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName(minioNamespace, "capacity", "total"),
```

Comment:
`minio_capacity_total` and `minio_capacity_usable` may be confusing, I propose `minio_capacity_raw` and `minio_capacity_usable`. 

Similarly `minio_capacity_raw_free` and `minio_capacity_usable_free`

Reply:
This makes sense. Will do.